andela/ah-frontend-odin
# #161340154 Enable users to bookmark articles for reading later

#### What does this PR do?
Implements bookmarking articles
#### Description of Task to be completed?
Enable users to bookmark articles for reading later
#### How should this be manually tested?
- git pull branch `ft-bookmarking-articles-161340154` and check it out
- Run server using `yarn start` command
- Login and bookmark an article
#### What are the relevant pivotal tracker stories?
[#161340154](https://www.pivotaltracker.com/story/show/161340154)
#### Screenshots 
<img width="1416" alt="screen shot 2018-12-19 at 21 24 36" src="https://user-images.githubusercontent.com/39129473/50239924-87ee4400-03d4-11e9-9506-3d6c87ea6645.png">
